misk-slack/src/main/kotlin/slack-api/FakeSlackApi.kt

```diff
@@ -0,0 +1,20 @@
+package `slack-api`
```

Comment:
Hmmm this isn't really that ideal or idiomatic.

How about `misk.slack`? Backticks or hyphens in package names are not common or best practice. 

For example, the Misk integration with AWS Dynamo is `misk.dynamodb` https://github.com/cashapp/misk/blob/c997e2dfeadf24c46b21153a830f8bd4f2278c7a/misk-aws-dynamodb/src/main/kotlin/misk/dynamodb/DynamoDbService.kt#L1